src/components/nodes-table/index.vue

```diff
@@ -240,7 +240,12 @@
 						item.firmwareUpdate.progress
 					}}</span>
 				</v-progress-circular>
-				<div v-else>v{{ item.firmwareVersion }}</div>
+				<div v-else>
+					v{{
+						item.firmwareVersion +
+						(item.sdkVersion ? ` (${item.sdkVersion})` : '')
```

Comment:
1.: `v` before the firmware version, but no `v` before the SDK version?
2.: I would clearly indicate which is which. firmware version and sdk version have no correlation on 500 series sticks. They do on 700 series sticks with stock firmware, but for Razberry I'd expect them to be different too - they compile their own.

How about:
```
FW: vA.B
SDK: vX.Y.Z
```